CHANGELOG.md

```diff
@@ -41,6 +41,9 @@
   [Allen Zeng](https://github.com/allen-zeng)
   [#1175](https://github.com/realm/SwiftLint/issues/1175)
 
+* Fix `redundant_void_return` matches if return type starts with Void~.
```

Comment:
You need to add two trailing spaces as described in https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes